src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -391,4 +481,44 @@ void Src_End_Deleptonization()
 
 
 
-#endif // #if ( MODEL == HYDRO )
+
+//-----------------------------------------------------------------------------------------
+// Function    :  YeOfRhoFunc
+// Description :  Calculate electron fraction Ye from the density
+//
+// Note        :  1. Invoked by Src_Deleptonization()
+//                2. Add "#ifndef __CUDACC__" since this routine is only useful on CPU
```

Comment:
Is the description here true?

Reply:
Yes right. Anyway, I slightly changed the description.

Reply:
My bad. I mean the description in Note (2).

Reply:
I think Note (2) is unnecessary. I removed this.